src/geom/sides.rs

```diff
@@ -15,6 +15,111 @@ pub struct Sides<T> {
     pub bottom: T,
 }
 
+#[derive(Debug, Default, Copy, Clone, Eq, PartialEq, Hash)]
+pub struct Margin {
```

Comment:
I believe this struct should be in `page.rs` (it isn't used at all by the Typst base crate).

Reply:
Done.